packages/mdc-chips/chip/foundation.js

```diff
@@ -107,6 +112,25 @@ class MDCChipFoundation extends MDCFoundation {
     }
   }
 
+  /**
+   * Handles a transition end event on the root element.
+   * This is a proxy for handling a transition end event on the leading icon or checkmark,
+   * since the transition end event bubbles.
+   * @param {!Event} evt
+   */
+  handleTransitionEnd_(evt) {
+    if (!(evt.propertyName === 'opacity')) {
```

Comment:
Ease up on the parens... `if (evt.propertyName !== 'opacity')`

Reply:
Wow major brain fart on the opposite of ===  😛